### DISCUSSION TITLE
Starting a discord group?

### DISCUSSION THREAD

**Discussion by lostdesign** [pimcore/pimcore#8498] [Ideas]
Discussions here make sense for any thread like things, but for any quick questions / support kinda things, wouldn't be nicer to have a discord chat for that?

I do know that the gitter exists but the UX there really isn't really helpful at all I'd say. With discord you can structure things in channels and rely on a "unread" and "reply to" feature, much more than gitter does it.

I'd be up for starting that group and maintaining it, if pimcore cannot support another platform, but would love if we can decide to move to a different place.

@fashxp @brusch

**Comment by dpfaffenbauer** [comment]
we've had that discussion before and I proposed slack... still most of the people thought that gitter is good enough.

**Reply by lostdesign** [reply]
Slack would be good too. Surely people may find gitter fine, getting into it is easy since it connects to your github.

Discords way of getting in is also rather easy as you are not forced to create and account as opposed to slack.

Discord would be free too, I'm more about a better UX and management of a chat, which gitter cannot offer.

**Comment by jeykodev** [comment]
+1 for that. I Think one of the most important thing to concider is a solid search function. In 90% of the cases a question was already asked, but most times it is hard to find and you have to go trough many different channels (Stackoverflow, Github Issues, Github Community, old Forum, Gitter).

**Comment by danielehrhardt** [comment]
Something new here?

**Comment by vmvartiainen** [comment]
Looks like gitter is dead. Is there a slack workspace?

**Comment by drenault-delko** [comment]
I'm prefered Slack :smiley:

**Reply by AlephLau** [reply]
Me too

**Comment by Gitman007** [comment]
I personally hate anything that is hidden from the (searchable) open web for something like this. 
Loads of valuable information that especially newcomers appreciate, will not be found anymore.

**Reply by AlephLau** [reply]
I understand your point @Gitman007 , but in the case of niche technologies like Pimcore, having a more active and connected community like what Slack offers can be a big advantage over scattered GitHub Discussions. In the end, anyone truly interested in Pimcore will likely come across the Slack community and find the support or information they need there. I don’t think it’s necessarily a disadvantage just a different way of building a more engaged ecosystem.

**Reply by drietsch** [reply]
@Gitman007 Totally agree — information that's hidden from the open web can be a real barrier, especially for newcomers who rely on searchable resources to get started and find support.

@AlephCode Thanks for your input as well!

May I also involve the Pimcore Open Community Alliance (POCA) in this discussion? We have our next meeting next week, and I believe this topic — especially around communication and accessibility — is important to clarify or even actively support. Would it be okay if I connected you with some of our folks from POCA to explore this further?

Cheers,
– Dietz

**Reply by BerndDultinger** [reply]
I´ll pick up for POCA agenda

**Reply by jeykodev** [reply]
@drietsch Great to see things finally moving forward here after four years! A lot has happened since then... If we keep building an open platform for exchange (that can be indexed) we might actually reach the point where some AI can solve all our Pimcore problems. 😄 

Imagine how many hours we've all spent in the past trying to explain to people why the save button is greyed out. This is the kind of tedious task I'd be happy to let the Pimcore Support AI handle for me. ;)
